README.md

```diff
@@ -8,8 +8,7 @@ your git repositories.
 ## Install
 
 ```
-$ curl https://raw.githubusercontent.com/joshfriend/hooker/master/hooker.sh
-$ ./hooker.sh --install
+$ bash <(curl -s https://raw.githubusercontent.com/joshfriend/hooker/master/hooker.sh) --install
```

Comment:
I don't think this would work since the script aliases itself to all the git hook files in `.git/hooks` so that when a git hook is run, `hooker.sh` is executed and then finds the right hooks to run in `.hooks`.